src/GovernorCountingFractional.sol

```diff
@@ -250,96 +258,4 @@ abstract contract GovernorCountingFractional is Governor {
         }
     }
 
-    /**
-     * @notice Cast a vote with a reason and additional encoded parameters using
-     * the user's cryptographic signature.
-     *
-     * Emits a {VoteCast} or {VoteCastWithParams} event depending on the length
-     * of params.
-     *
-     * @dev If casting a fractional vote via `params`, the voter's current nonce
-     * must be appended to the `params` as the last 16 bytes and included in the
-     * signature. I.e., the params used when constructing the signature would be:
-     *
-     *   abi.encodePacked(againstVotes, forVotes, abstainVotes, nonce)
-     *
-     * See {fractionalVoteNonce} and {_castVote} for more information.
-     */
-    function castVoteWithReasonAndParamsBySig(
```

Comment:
This was added because the v4 version did not handle replay attacks. I think the v5 uses a nonce now so we don't need to override it.

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/920225a1c7e6e749ba23b10ed076b04f454290e8/contracts/governance/Governor.sol#L577